fsh-tank/input/fsh/CervicalCytologyResult.fsh

```diff
@@ -42,6 +42,34 @@ Usage: #definition
 * compose.include[=].concept[=].display = "SCC"
 * compose.include[=].concept[=].designation.use = $SNOMED#900000000000003001 "Fully specified name"
 * compose.include[=].concept[=].designation.value = "Squamous cell carcinoma (morphologic abnormality)"
+* compose.include[=].concept[+].code = #373882004
```

Comment:
Same comments regarding the codes added to the `CytologyInterpretation` function apply to the codes added here.

Reply:
See https://fhir-ru.github.io/valueset.html. You should be able to add a new code system to ValueSet.compose.include in the fsh-tank/input/fsh/CervicalCytologyResult.fsh file.

Reply:
As a result of a recent conversation on Slack, we will plan to leave the LOCAL codes out for now. These can be added if needed when we expand upon our abilities to enter new data into the Dashboard or CDS Hooks apps.